tests/Microsoft.Identity.Test.Integration.net45/Microsoft.Identity.Test.Integration.net45.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net462</TargetFramework>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
We have a certificate signing method that behaves differently on >4.6.2 and 4.7+. This change will allow our certificate tests to run on 4.7 and hit the new api for testing. Is this ok or should I create an additional project that targets 4.7 and keep this one at 4.6.2.

Reply:
@henrik-me @bgavrilMS @MarkZuber

Reply:
If we have code paths that behave differently on .net46 and .net47, we should have them both tested, i.e. create a simple test project that targets net46 with the tests that go through that code path and keep net47 for all tests.